sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -41,3 +41,12 @@ select max(cast(v as interval)) from VALUES ('1 seconds'), ('4 seconds'), ('3 se
 
 -- min
 select min(cast(v as interval)) from VALUES ('1 seconds'), ('4 seconds'), ('3 seconds') t(v);
+
+-- SPARK-29605: cast string to intervals
```

Comment:
This should be `SPARK-29822` because this is newly added test coverage for this PR `SPARK-29822`.

Reply:
I guess I just rm it from here, thanks